apps/fz_http/lib/fz_http/api_tokens.ex

```diff
@@ -24,13 +24,9 @@ defmodule FzHttp.ApiTokens do
 
   def get_api_token!(id), do: Repo.get!(ApiToken, id)
 
-  def get_unexpired_api_token(api_token_id) do
-    now = DateTime.utc_now()
-
-    Repo.one(
-      from a in ApiToken,
-        where: a.id == ^api_token_id and a.expires_at >= ^now
-    )
+  def fetch_unexpired_api_token_by_id(api_token_id, now \\ DateTime.utc_now()) do
+    from(a in ApiToken, where: a.id == ^api_token_id and a.expires_at >= ^now)
+    |> Repo.fetch()
```

Comment:
I think it's better to avoid exposing `now` attribute for testing, instead we can create tokens with a custom `expires_at` using fixtures. Even though it's innocent in this case, in general, it leads to functions with a lot of parameters that exist only for test env and makes code not readable. We will fixtures anyways for e2e tests, etc.
```suggestion
  def fetch_unexpired_api_token_by_id(api_token_id) do
    from(a in ApiToken, where: a.id == ^api_token_id and a.expires_at >= fragment("NOW()"))
    |> Repo.fetch()
```